README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
```

Comment:
The README.md file needs to be professional

Reply:
- [ ] Well, this was a point but I think it can be done even better, as a code reviewer,  p please always communicate with the student in a polite and friendly way, remember, you should not be imperative in your comment, instead, use suggestions, this way the student will understand that he/she did something, okay but that may be improved. in addition, why the student should agree to your changes? well, this is why you need to explain to the student why the changes you have requested are important to contribute to his professionalism and following some code or repository best practices. 
- [ ] Make use of emojis in your comments to sound a bit more friendly.
- [ HINT-EXAMPLE ] - [ ] Well done on using the readme file, however, I would suggest you personalize it with your project details, this will allow anyone visiting your work to understand what it stands for before they even read the code, the readme file should be as descriptive as possible. what do you think if you make a few touches to improve it? :🥰

Reply:
Thanks for the details instructions .